assets/src/dashboard/animations/parts/index.js

```diff
@@ -70,6 +71,7 @@ export function AnimationPart(type, args) {
       [ANIMATION_TYPES.MOVE]: AnimationMove,
       [ANIMATION_TYPES.SPIN]: AnimationSpin,
       [ANIMATION_TYPES.ZOOM]: AnimationZoom,
+      [ANIMATION_EFFECTS.FADE_IN]: EffectFadeIn,
```

Comment:
I figured it'd be cleanest to just include `ANIMATION_EFFECTS` in our animation parts lookup since their signature will be the same and all our existing code... just works.

Reply:
Hmmm... I'll see if I can come up with some way of testing these effects so they'd at least throw an error if the primitive animations they depend upon suddenly change (maybe a snapshot type test).  This way if we do update a primitive for any reason, we'll be forced to double check all the effect that rely on it to make sure nothing was broken in the process.